stdlib/src/array.rs

```diff
@@ -1258,9 +1258,11 @@ mod array {
 
     impl Iterable for PyArray {
         fn iter(zelf: PyRef<Self>, vm: &VirtualMachine) -> PyResult {
+            let zelf_ = zelf.clone();
```

Comment:
`_` prefix usually means unused variable in rust code.

Reply:
The `_` _suffix_ usually indicates that you're dodging the use of a reserved keyword. From the [Rust style guide](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/advice.md):

> When a name is forbidden because it is a reserved word (e.g., `crate`), use a trailing underscore to make the name legal (e.g., `crate_`), or use raw identifiers if possible.